### ISSUE TITLE
a11y: Curl command's copy to clipboard button has accessibility issues

### ISSUE THREAD

**Issue by ghost** [swagger-api/swagger-ui#9584]
### Q&A (please complete the following information)
 - OS: [e.g. macOS]  Windows 10
 - Browser: [e.g. chrome, safari]  Chrome
 - Version: [e.g. 22]  121
 - Method of installation: [e.g. npm, dist assets]  npm
 - Swagger-UI version: [e.g. 3.10.0]  Swagger UI React 5.11.3
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0]  OpenAPI 3.0

### Content & configuration
Basic/ default

### Describe the bug you're encountering
We are using a tool called Sort Site to evaluate our site for 508 accessibility compliance.  It is flagging for the "copy to clipboard" element that "This "button" element is empty and has no accessible name.  A programmatically determined name allows screen readers to tell users what the control does."
Violates WCAG 2.0 A 4.1.2  https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
This error is on the button (with no class) which is contained in the div with class "copy-to-clipboard"

### To reproduce...
Open the pane for a method such as POST.  Click Try It Out.  Click Execute.  Scroll down to see the Curl command - there is a small Copy to Clipboard icon in the bottom right corner.

### Expected behavior
The button should identify itself

### Screenshots
Here's the area on the screen
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/6b15f4c3-a045-47ff-b5a4-71d2f2a7bfb9)

Here's the Sort Site error detail:
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/f3503f3e-9b38-4870-a045-c844afa98e89)

**Comment by mgifford** [comment]
The button definitely needs a semantically defined name.

**Comment by moskvin** [comment]
I've opened PR #10908 which addresses this. If you're affected by this issue, an upvote or comment there might help it get reviewed. 🙏
